instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.28b1"
+__version__ = "0.28b2"
```

Comment:
You don't need to modify this, maintainers will bump it when they release a new version

Reply:
Reverted the changes